server/storage/schema/auth_roles.go

```diff
@@ -32,6 +32,40 @@ func (abe *authBackend) GetRole(roleName string) *authpb.Role {
 }
 
 func (atx *authBatchTx) UnsafeGetRole(roleName string) *authpb.Role {
```

Comment:
Any reason to reorder functions? It might make backport harder.

Reply:
I did not reorder the methods, instead i just added some new methods. It's just a github displaying issue.

Reply:
ack